flexget/plugins/output/rss.py

```diff
@@ -198,7 +198,7 @@ def on_task_exit(self, task, config):
                 )
                 rss.title = entry['title']
             for field in config['link']:
-                if field in entry:
+                if field in entry and entry[field] is not None:
```

Comment:
```suggestion
                if entry.get(field) is not None:
```
This is probably the standard way to check both at once.